pkgs/top-level/perl-packages.nix

```diff
@@ -4481,6 +4481,19 @@ let
     propagatedBuildInputs = [ Clone ];
   };
 
+  CSSMinifier = buildPerlPackage {
+    pname = "CSS-Minifier";
+    version = "0.01";
+    src = fetchurl {
+      url = "mirror://cpan/authors/id/P/PM/PMICHAUX/CSS-Minifier-0.01.tar.gz";
+      sha256 = "0Kk0m46LfoOrcM+IVM+7Qv8pwfbHyCmPIlfdIaoMf+8=";
+    };
+    meta = {
+      description = "Perl extension for minifying CSS";
+      license = lib.licenses.artistic1;
```

Comment:
Missing maintainer

Reply:
It looks like some / most packages in here don't have maintainers, so I figured that was the norm -- happy to add us if that's not the case, however.